modules/subscription/modules/subscription/README.md

```diff
@@ -0,0 +1,3 @@
+# Subscriptions Module
+
+## Dependancy -> Payment Module
```

Comment:
Could you please detail any manual steps required, if any?

Please also add a brief description of the module. And you can copy that description into the package.json description too.